wire-ios/Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController+Files.swift

```diff
@@ -94,13 +94,17 @@ extension ConversationInputBarViewController {
 
             guard let self else { return }
 
-            impactFeedbackGenerator.prepare()
-            ZMUserSession.shared()?.perform {
+            self.impactFeedbackGenerator.prepare()
```

Comment:
```suggestion
            impactFeedbackGenerator.prepare()
```

I would remove `self` for constistency with `userSession` below